src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatenotlistable/UpdateNotListableAboutToSubmitHandler.java

```diff
@@ -47,6 +48,10 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
             sscsCaseData.setDirectionDueDate(sscsCaseData.getUpdateNotListableDueDate());
         }
 
+        if ("no".equalsIgnoreCase(sscsCaseData.getUpdateNotListableSetNewDueDate())) {
```

Comment:
Are you sure you want to clear the field, when they don't want to set a new due date? I thought they would want to retain the existing due date?